CONTRIBUTING.md

```diff
@@ -52,35 +52,36 @@ Once you have a GitHub login:
 
 2. Then start a terminal (or use your favorite git GUI app) and clone your fork of the Exiv2 repo:
 
-        $  git clone https://github.com:your-username/exiv2.git
+        $ git clone https://github.com:your-username/exiv2.git
+        $ cd exiv2
+
+3. Set (add) the remote repository:
+
+        $ git remote add exiv2_upstream https://github.com/Exiv2/exiv2.git
+		$ git fetch exiv2_upstream master
```

Comment:
This step would not be needed to add the remote. We can add a remote without needing to fetch the data from it inmediately. I would add this line in a separate point:

```
3. Add a new remote pointing to upstream
...
4. Fetch data from upstream
$ git fetch exiv2_upstream master
```

Reply:
This was needed for the `git checkout exiv2_upstream master` stuff to work - right @D4N ?

Reply:
Yes, it is needed for that. What I meant is that it can be a bit confusing, since it does nothing to do with adding a remote to your local git configuration (that's why it is called `git fetch` and not `git remote` :wink: ). But it is a minor detail. 

By the way, I was confused by your profile picture and I thought it was @D4N the one commiting those changes hehe. I'm sorry for the confusion.

Reply:
I moved the `$ git fetch exiv2_upstream master` command to "4." and reordered the text a bit. I hope this make things more clear.